shared/productTypes.tsx

```diff
@@ -219,6 +221,9 @@ const calculateProductTitle =
 	(baseProductTitle: string) => (mainPlan?: SubscriptionPlan) =>
 		baseProductTitle + (mainPlan?.name ? ` - ${mainPlan.name}` : '');
 
+const calculateSupporterPlusTitle = (interval: string) =>
+	interval === 'month' ? 'monthly + extras' : 'annual + extras';
```

Comment:
this could probably be a little shorter:

`\`${interval === 'month' ? 'monthly' : 'annual'} + extras\``

Reply:
It was done this way because 'monthly + extras' and 'annual + extras' are static strings of the ProductFriendlyName type. If we try interpolate the string we get a type error.